### PR TITLE
New package: cre2-0.3.1

### PR DESCRIPTION
**Pull request by okapia**
C language wrapper for RE2 the regular expressions library

### CODE REVIEW

File: srcpkgs/cre2/template [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'cre2'
+pkgname=cre2
+version=0.3.1
+revision=1
+build_style="gnu-configure"
+hostmakedepends="re2 autoconf automake libtool perl"
```
Comment (Vaelatern) [inline srcpkgs/cre2/template]: Autoconf redundant. Is `re2` actually run in the building of this package?

File: srcpkgs/cre2/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+makedepends="re2-devel"
+short_desc="C language wrapper for RE2 the regular expressions library"
+maintainer="Oliver Kiddle <okiddle@yahoo.co.uk>"
+license="3-clause-BSD"
+homepage="https://github.com/marcomaggi/cre2/"
```
Comment (Vaelatern) [inline srcpkgs/cre2/template]: Ideally remove slash at end of url

File: srcpkgs/cre2/template [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum="48a326ce633ef35b4c138681e898d7676a3b7a17d9eb5b29994a34313c19d299"
+
+CXXFLAGS="-std=c++11"
+
+pre_configure() {
+	sh ./autogen.sh
```
Comment (Vaelatern) [inline srcpkgs/cre2/template]: Can't you just `./autogen.sh`?

Comment (okapia) [inline srcpkgs/cre2/template]: No, not unless I do chmod +x on it first.

File: srcpkgs/cre2/template (hunk 2 above)
Comment (okapia) [inline srcpkgs/cre2/template]: done.

File: srcpkgs/cre2/template (hunk 1 above)
Comment (okapia) [inline srcpkgs/cre2/template]: I've removed both.

File: srcpkgs/cre2/template (hunk 3 above)
Comment (Vaelatern) [inline srcpkgs/cre2/template]: Does it not just do `autoreconf -fi`?

**Comment by Vaelatern** [comment]
There is no need to close and reopen PRs, just for the record.

File: srcpkgs/cre2/template (hunk 3 above)
Comment (okapia) [inline srcpkgs/cre2/template]: No, there's a mkdir and libtoolize and slightly different autoreconf options.

**Comment by okapia** [comment]
For what it's worth, I didn't manually close it. Network fell out during an attempt to push the update and github somehow decided that meant I wanted to close it.

**Comment by Vaelatern** [comment]
Thank you for not being intentional.

**Pull request closed without merging**